autotx/utils/ethereum/constants.py

```diff
@@ -1,7 +1,7 @@
 MASTER_COPY_ADDRESS = "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552"
-MASTER_COPY_L2_ADDRESS = "0xfb1bffC9d739B8D520DaF37dF666da4C687191EA"
+MASTER_COPY_L2_ADDRESS = "0x3E5c63644E683549055b9Be8653de26E0B4CD36E"
```

Comment:
what if we make `MASTER_COPY_L2_ADDRESS(ES)` a list and add both values? (the one you added and the one we previous had) - This way we guarantee that we can interact with safe in all the networks we support (the previous value is needed for optimism or sepolia)